### ISSUE TITLE
All Winget Packages Failing

### ISSUE THREAD

**Issue by GeneralGarcia** [microsoft/winget-cli#3410]
### Brief description of your issue

It seems all my Winget deployments via Intune are failing this morning with the following error: **0x8a15000f : Data required by the source is missing**

I can replicate this by trying to install any Winget application manually via the command line, which throws the same error.

I have tried **winget reset source** but that doesn't make any difference. 

Is there anything I can do? Or is this a CDN issue or similar?

### Steps to reproduce

Try any winget install via the command line, they all throw the same error

### Expected behavior

Installs as normal

### Actual behavior

Failed when searching source: winget
An unexpected error occurred while executing the command:
0x8a15000f : Data required by the source is missing

### Environment

```shell
Windows Package Manager v1.4.11071
Windows: Windows.Desktop v10.0.22621.1848
Package: Microsoft.DesktopAppInstaller v1.19.11071.0
```

**Comment by evanjs** [comment]
I am experiencing the same issue with different symptoms/messages.
This is what I get when attempting to update:
```
Failed when searching source: msstore
An unexpected error occurred while executing the command:
0x8a15005e : The server certificate did not match any of the expected values.
```

Inspecting the certificate that seems to be associated with the endpoints being hit shows some... obviously sketches sites listed under the Subject Alternative Name, which cURL notes is the reason for the failure

**Comment by evanjs** [comment]
> Can you download the MSIXBundle from https://aka.ms/getwinget and manually install it? Possibly remove it by using the Store and reinstall from the MSIXBundle?

Everything appears to work as intended after upgrading via https://aka.ms/getwinget.
Hopefully OP has the same experience.
___
### Environment
```
Windows Package Manager v1.5.1572
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22631.1972
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.20.1572.0
```

**Comment by denelon** [comment]
WinGet 1.4 originally had a certificate root for the "msstore" source that was deprecated.
We did have a 1.4 servicing release with the new certificate.
https://github.com/microsoft/winget-cli/releases/tag/v1.4.11071

WinGet 1.5 is now out and also has the correct certificate.
https://github.com/microsoft/winget-cli/releases/tag/v1.5.1572
We've updated the https://aka.ms/getwinget link to point to the latest stable release.

**Comment by denelon** [comment]
@GeneralGarcia are you still having this issue?

**Comment by GeneralGarcia** [comment]
Thanks for all the suggestions on this. I'm in the UK so not in the office
currently but I'll test first thing tomorrow morning.

**Comment by GeneralGarcia** [comment]
Unfortunately still the same this morning :(

I've updated to the following version on two seperate machines: **Windows Package Manager v1.5.1572**

I then tried running a few install commands via an admin powershell window, such as: **winget install -e --id zxch3n.PomodoroLogger** or **winget install -e --id SleuthKit.Autopsy**

It doesn't seem to matter which package I try, I just get the following on both PCs:
**Failed when searching source: winget
An unexpected error occurred while executing the command:
0x8a15000f : Data required by the source is missing**

I tried the **winget source reset --force** command on both machines, and it re-prompted me to accept the agreements when I ran the install command, but with the same error at the end unfortunately. Not sure where to go from here? All was working fine until Wednesday.
